build.gradle.kts

```diff
@@ -14,6 +14,12 @@ allprojects {
         mavenCentral()
     }
 
+    dependencies {
+        constraints {
+            add("implementation", "org.jspecify:jspecify:1.0.0")
```

Comment:
```suggestion
            implementation("org.jspecify:jspecify:1.0.0")
```

Reply:
after the latest rebase `implementation` isn't known in the root build.gradle.kts anymore, as the java-library plugin is only applied in the module gradle config files - so i just specified the version directly for now.

Reply:
ah. maybe version catalog would be the way to go https://docs.gradle.org/current/userguide/centralizing_dependencies.html#sub:using-catalogs